src/templates/tag.js

```diff
@@ -3,6 +3,7 @@ import { Helmet } from 'react-helmet';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 import PropTypes from 'prop-types';
+import queryString from 'query-string';
```

Comment:
Could we swap the `query-string` library use with the `parseQueryString` function in `src/utils/query-string.js` for both?